darts/models/forecasting/exponential_smoothing.py

```diff
@@ -34,7 +34,10 @@ def __init__(
         <https://www.statsmodels.org/stable/generated/statsmodels.tsa.holtwinters.ExponentialSmoothing.html>`_;
         we refer to this link for the original and more complete documentation of the parameters.
 
-        `model_mode` must be a ``ModelMode`` Enum member. You can access the Enum with ``from darts import ModelMode``.
+        `model_mode` must be a ``ModelMode`` Enum member. You can access the Enum with
```

Comment:
Do we still need this line?

Reply:
Yes would say so as the ModelMode enum is still used for the trend argument in the ES model

Reply:
Then I think we should write `trend` must be a ``ModelMode`` ... no?

Reply:
Jup of course - apologies for being sloppy here. Fixed it now.